internal/worker/sort.go

```diff
@@ -24,10 +24,29 @@ func (a ByTimeCreated) Less(i, j int) bool {
 	}
 
 	return inum < jnum
+}
+
+// ByTimeUploaded implements sort.Interface for []Digest based on
+// the TimeUploadedMs field.
+type ByTimeUploaded []Digest
+
+func (a ByTimeUploaded) Len() int      { return len(a) }
+func (a ByTimeUploaded) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a ByTimeUploaded) Less(i, j int) bool {
+	inum, err := strconv.Atoi(a[i].TimeUploadedMs)
+	if err != nil {
+		log.Fatal(err)
+	}
 
+	jnum, err := strconv.Atoi(a[j].TimeUploadedMs)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return inum < jnum
```

Comment:
To keep newer tag, I think this should be

```
return inum > jnum
```

Reply:
You sure? I'll write some unit tests just to be sure :))

Reply:
I think you are right! I fixed it and wrote some tests. Thanks for the help!